### PR TITLE
Bug 1988210: Bump helm to v3.6.2

### PR DESCRIPTION
**Pull request by jhadvig**
/assign @spadgett 

ccing @dperaza4dustbit

### CODE REVIEW

File: vendor/OWNERS [hunk 1]
```diff
@@ -1,8 +0,0 @@
-reviewers:
```
Comment (dperaza4dustbit) [inline vendor/OWNERS:1]: Is this file supposed to be deleted?

File: vendor/OWNERS [hunk 2]
```diff
@@ -1,8 +1,9 @@
+
 reviewers:
   - florkbr
   - jhadvig
 approvers:
   - florkbr
 labels:
   - component/backend
-  - kind/dependency-change
+  - kind/dependency-change
```
Comment (dperaza4dustbit) [inline vendor/OWNERS:9]: Add the new line?

**Review by dperaza4dustbit — APPROVED** [review]

**Comment by dperaza4dustbit** [comment]
This change is not required in 4.8 but will be nice to have. On 4.7 and 4.6 I would not take the risk since there is no security CVE effecting HELM when it integrates with console. See my comments here: https://bugzilla.redhat.com/show_bug.cgi?id=1984588

**Comment by jhadvig** [comment]
It seems that the `generated` field in the `chartproxy/testdata/` are holding incorrect timestamps. @dperaza4dustbit could you please assist on that. Thanks!

**Comment by spadgett** [comment]
@jhadvig nit: you have a typo in the commit message

**Comment by dperaza4dustbit** [comment]
@jhadvig feel free to cherry pick PR 9695 here. Will be easier to integrate.

**Comment by jhadvig** [comment]
@dperaza4dustbit commit cherry-picked ✅ 
@spadgett commit message fixed ✅ 

PTAL

**Comment by dperaza4dustbit** [comment]
lgtm

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**